### ISSUE TITLE
Clicking on 'pending' does not prevent double payments

### ISSUE THREAD

**Issue by ekzyis** [stackernews/stacker.news#1672]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When you create an item and the wallet is not immediately able to pay it, we show `pending` next to it.

When this is clicked, we show a QR code for the same invoice we're currently still attempting to pay in the background via the attached wallets.

This means it is possible that the invoice can get paid twice.

### Steps To Reproduce

1. Attach wallet that doesn't respond
2. Create item
3. Click on pending
4. Check console: we're still polling for the same invoice in the background

### Expected behavior

Clicking on `pending` cancels the previous invoice and shows a new invoice.

**Comment by huumn** [comment]
> Clicking on pending cancels the previous invoice and shows a new invoice.

I don't think this is the expected behavior in all cases. Sometimes I click on pending to see how long the attached wallet has been attempting payment.

I agree with the general problem though. The current UI/UX allows double payments and it shouldn't.

**Comment by huumn** [comment]
That number doesn't update if I haven't navigated back to it. I'd probably still open anyway because I'd have to know/trust those numbers were correlated (and certainly the average stacker wouldn't know that).

**Comment by ekzyis** [comment]
> The current UI/UX allows double payments and it shouldn't.

Maybe we should only stop attempting the wallets in the background but keep the same invoice. Once the QR code is closed, we can continue to attempt payment with the wallets again.

**Comment by huumn** [comment]
There's no way to stop the wallets in the background generically, e.g. if I send off an NWC `pay_invoice` request or whatever it's called.

**Comment by ekzyis** [comment]
Ahh, right
